codec/decoder/core/src/mv_pred.cpp

```diff
@@ -320,6 +320,10 @@ int32_t GetColocatedMb (PWelsDecoderContext pCtx, MbType& mbType, SubMbType& sub
   }
 
   MbType coloc_mbType = colocPic->pMbType[iMbXy];
+  if (coloc_mbType == MB_TYPE_SKIP) {
+    //This indicates the colocated MB is P SKIP MB
+    coloc_mbType |= MB_TYPE_16x16 | MB_TYPE_P0L0 | MB_TYPE_P1L0;
+  }
```

Comment:
This change doesn't seems like a general fix, not CAVLC specific. It should be in a separate commit?